src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -959,6 +960,65 @@ def _configure_security_policy_storage_params(arg_ctx):
         c.argument('allow_data_loss',
                    arg_type=allow_data_loss_param_type)
 
+    ###############################################
+    #             sql instance pool               #
+    ###############################################
+
+    with self.argument_context('sql instance-pool') as c:
+        c.argument('instance_pool_name',
+                   options_list=['--name', '-n'],
+                   help="Instance Pool Name")
+
+        c.argument(
+            'tier',
+            arg_type=tier_param_type,
+            required=True,
+            help='The edition component of the sku. Allowed value: GeneralPurpose.')
+
+        c.argument('family',
+                   arg_type=family_param_type,
+                   required=True,
+                   help='The compute generation component of the sku. '
+                   'Allowed value: Gen5')
+
+    with self.argument_context('sql instance-pool create') as c:
+        # Create args that will be used to build up the InstancePool object
+        create_args_for_complex_type(
+            c, 'parameters', InstancePool, [
+                'location',
+                'license_type',
```

Comment:
Try to use get_enum_type for --license-type for allowed values

Reply:
Fixed

Reply:
Fixed